.github/workflows/main.yml

```diff
@@ -31,5 +31,10 @@ jobs:
         with:
           node-version: ${{ matrix.node }}
 
+      - name: Before Install
+        run: |
+          git config --global user.name "Travis CI"
```

Comment:
"Cleanup travis" ?  :wink:

Reply:
🤦 Wonder why this never went through my head. Maybe you guys wanna just update this name directly in master? 😄